src/test/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/service/KalendarServiceTest.java

```diff
@@ -69,6 +91,47 @@ public void setKalendarAttribute_customNameIsNull() {
     assertTrue(customName, actualKalendar.getName().length() > 0);
   }
 
+  @Test
+  public void deleteGoogleCalendar_wrongClientToken() {
+    String expectedMessage = "No such user in the database";
+    doThrow(new NullPointerException()).when(kalUserRepository).findByClientToken(anyString());
+
+    try {
+      kalendarService.deleteKalendar("",1l);
+    } catch (ValidationException expected) {
+      assertEquals(expectedMessage, expected.getMessage());
```

Comment:
this test fails for me with this error:
`org.junit.ComparisonFailure: 

Expected :No such user in the database
Actual   :No such kalendar in the database`